ui/app/src/components/PublishOnDemandWidget/PublishOnDemandWidget.tsx

```diff
@@ -201,7 +201,7 @@ export function PublishOnDemandWidget(props: PublishOnDemandWidgetProps) {
             message: formatMessage(messages.publishSuccess)
           })
         );
-        setPublishGitFormData(initialPublishGitFormData);
+        setPublishGitFormData({ ...initialPublishGitFormData, environment });
```

Comment:
Can we rename to `publishingTarget` with ease?

Reply:
done!